modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -106,7 +106,45 @@ definition {
 		}
 	}
 
-	@description = "LPS-180070 - Verify creating a Commerce Product entry triggers an action to add a user"
+	@description = "LPS-173537 - Verify creating an Account entry triggers an action to add a second Account entry"
+	@priority = 5
+	test CanAddAccountEntryAfterCreatingAccountEntry {
+		property portal.acceptance = "true";
+
+		task ("Given an Account object") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectSystemObject(label = "Account");
+		}
```

Comment:
We don't need to create a separate step for this. We can just include this while adding the action.